mix.exs

```diff
@@ -9,14 +9,15 @@ defmodule DateTimeParser.MixProject do
       version: @version,
       homepage_url: "https://hexdocs.pm/date_time_parser",
       source_url: "https://github.com/taxjar/date_time_parser",
-      elixir: ">= 1.3.0",
+      elixir: ">= 1.4.0",
```

Comment:
Turns out some of the libraries we use require `1.4+` so I have to bump this up too.